packages/care-package/core/src/CarePackage.ts

```diff
@@ -204,8 +211,62 @@ export class CarePackage {
         const content = JSON.stringify(manifest, null, 2);
         // TODO extract 'manifest.json' as a const variable
         zip.addFile('manifest.json', Buffer.alloc(content.length, content));
-        zip.writeZip(path.join(dist, name));
-        console.log(`CARE package created: ${path.join(dist, name)}`);
+        const archivePath = path.join(dist, name);
+        zip.writeZip(archivePath);
+        console.log(`CARE package created: ${archivePath}`);
+
+        if (iso) {
+            this.ifPythonAvailable()
+                .then(() => this.createIsoArchive(archivePath, executable))
+                .catch(err => console.log(err));
+        }
+    }
+
+    async ifPythonAvailable(): Promise<void> {
```

Comment:
This should be much simpler way if you are not requiring certain version check. But your code is compatible both with 2 and 3, so...
var child = require('child_process');
var exists = child.spawnSync('which', ['python']).status === 0;